new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -573,20 +596,291 @@ public void testDomainInPvUsingWdt() {
+        "Creating WLST properties file failed");
+    Properties p1 = new Properties();
+    p1.setProperty("admin_host", K8S_NODEPORT_HOST);
+    p1.setProperty("admin_port", Integer.toString(t3ChannelPort));
+    p1.setProperty("admin_username", ADMIN_USERNAME_DEFAULT);
+    p1.setProperty("admin_password", ADMIN_PASSWORD_DEFAULT);
+    p1.setProperty("cluster_name", clusterName);
+    p1.setProperty("test_name", "change_server_count");
+    assertDoesNotThrow(() -> p1.store(new FileOutputStream(wlstPropertiesFile), "wlst properties file"),
+        "Failed to write the WLST properties to file");
+
+    // change the server count of the cluster by running online WLST
+    Path configScript = Paths.get(RESOURCE_DIR, "python-scripts", "introspect_version_script.py");
+    executeWLSTScript(configScript, wlstPropertiesFile.toPath(), introDomainNamespace);
+
+    // construct a patch string to update introspectVersion
+    StringBuffer patchStr = new StringBuffer("[{")
+        .append("\"op\": \"add\", ")
+        .append("\"path\": \"/spec/introspectVersion\", ")
+        .append("\"value\": \"2")
+        .append("\"}]");
+    logger.info("Patch String \n{0}", patchStr);
+
+    logger.info("Adding introspectVersion for domain {0} in namespace {1} using patch string: {2}",
+        domainUid, introDomainNamespace, patchStr.toString());
+
+    // patch the custom resource
+    V1Patch patch = new V1Patch(new String(patchStr));
+    patchDomainCustomResource(domainUid, introDomainNamespace, patch, V1Patch.PATCH_FORMAT_JSON_PATCH);
+
+    //verify the introspector pod is created and runs
+    logger.info("Verifying introspector pod is created, run and deleted");
+    String introspectPodName = domainUid + "-" + "introspect-domain-job";
+    checkPodExists(introspectPodName, domainUid, introDomainNamespace);
+    checkPodDoesNotExist(introspectPodName, domainUid, introDomainNamespace);
+
+    //verify the maximum and minimum cluster size is updated to expected values
+    withStandardRetryPolicy.conditionEvaluationListener(new ConditionEvaluationListener() {
+        @Override
+        public void conditionEvaluated(EvaluatedCondition condition) {
+          logger.info("Waiting for Domain.status.clusters.{0}.maximumReplicas to be 6", clusterName);
+        }
+      })
+        .until((Callable<Boolean>) () -> {
+          Domain res = TestActions.getDomainCustomResource(domainUid, introDomainNamespace);
+          return (res.getStatus().getClusters().get(0).getMaximumReplicas() == 6)
+              && (res.getStatus().getClusters().get(0).getMinimumReplicas() == 2);
+        }
+        );
+  }
+
+  /**
+   * Test server pods are rolled and updated when introspector run triggered by introSpectVersion.
+   * Updates the admin server listen port using online WLST.
+   * Patches the domain custom resource with introSpectVersion.
+   * Verifies the introspector runs and pods are rolled in rolling fashion.
```

Comment:
Description should be read as  "Verifies the introspector runs and pods are restated in a rolling  fashion"

Reply:
fixed